src/core_deku/ticket_handle.mli

```diff
@@ -0,0 +1,15 @@
+type t [@@deriving yojson, ord, eq]
```

Comment:
This file is is dead, along with `Ticket_transition_table`

Reply:
removed